roles/kube_nginx_ingress/files/nginx_ingress_values.yml

```diff
@@ -1,8 +1,14 @@
 controller:
+  image:
+    tag: "0.17.1"
```

Comment:
We'll need to monitor closely when this changes upstream (and make sure we don't keep overwriting when upstream is at, say, 0.18.6).

Reply:
we need to revert this as soon as the PR that introduce the 0.16+ image in nginx-ingress is in.